client/gutenberg/extensions/simple-payments/editor.js

```diff
@@ -20,7 +20,9 @@ registerBlockType( 'a8c/simple-payments', {
 	category: 'jetpack',
 
 	keywords: [
-		/** @TODO add keywords */
+		'simple',
+		'payment',
```

Comment:
Having "Simple payments" as one keyword should work, since space isn't disallowed and it's a product name.